### DISCUSSION TITLE
可以实现自定义的 html 插入主题中吗？

### DISCUSSION THREAD

**Discussion by youngxhui** [HEIGE-PCloud/DoIt#934] [Q&A] [answered]
想将谷歌广告的代码插入博客中，可以不改动源代码的前提下插入代码吗

**Comment by eallion** [comment, marked as answer]
比如你想插入在文章页，就把主题的 [`layouts/_default/single.html`](https://github.com/HEIGE-PCloud/DoIt/blob/main/layouts/_default/single.html) 复制到博客根目录的 `layout` 目录中，这个目录的优先级高。

**Reply by youngxhui** [reply]
是博客的 `layouts` 目录下
